app/models/label/LabelValidationTable.scala

```diff
@@ -86,9 +99,143 @@ object LabelValidationTable {
     )
   }
 
+  case class ValidationCountPerDay(date: String, count: Int)
+
   def save(label: LabelValidation): Int = db.withTransaction { implicit session =>
     val labelValidationId: Int =
       (validationLabels returning validationLabels.map(_.labelValidationId)) += label
     labelValidationId
   }
+
+  def getOwnValidationCounts: Map[String, Int] = db.withSession { implicit session =>
```

Comment:
Could you just make sure that there are comments for all the functions you've written? I realize you are writing code that is similar to other functions that _don't_ have comments, but we are trying to improve the quality of the codebase over time :grin: